.circleci/config.yml

```diff
@@ -13,8 +13,8 @@ jobs:
           keys:
             - air01-{{ checksum "Cargo.lock" }}
       - run: |
-          rustup toolchain install nightly-2022-02-24-x86_64-unknown-linux-gnu
-          rustup default nightly-2022-02-24-x86_64-unknown-linux-gnu
+          rustup toolchain install nightly-2022-03-20-x86_64-unknown-linux-gnu
+          rustup default nightly-2022-03-20-x86_64-unknown-linux-gnu
```

Comment:
Why have you change it?

Reply:
https://github.com/fluencelabs/aquavm/runs/5822870555?check_suite_focus=true 
One of the packets broken build again, changing rust version solved it

Reply:
Ah, I see, let's apply these changes so.